### DISCUSSION TITLE
zigbee2mqtt ignore configuration.yaml

### DISCUSSION THREAD

**Discussion by ae-11** [Koenkk/zigbee2mqtt#6690] [General]
hardware: Xiaomi Lumi Gateway v2 (ARMv7) with Zigate firmware for JN5169 snapshot-20210103
Zigbee2MQTT version 1.18.1 
OpenWRT 19.07.5

/etc/zigbee2mqtt/configuration.yaml

serial:
  port: /dev/ttymxc1
  adapter: zigate

start - debug:
Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyACM0'
at Znp. (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)

Why zigbee2mqtt attempt connect to undeclared /dev/ttyACM0?

**Comment by kristianheljas** [comment]
Hey,

You need to indent `port` and `àdapter` to set them under `serial` dictionary:
```yaml
serial:
  port: /dev/ttymxc1
  adapter: zigate
```
